test/IntegrationTests/FileVectorizer/test_paginatedfilelist.py

```diff
@@ -0,0 +1,254 @@
+import logging
+import os
+import unittest
+import requests
+import sys
+import uuid
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../")))
+from utils.api_utils import add_customer, create_test_token, create_token_without_org_id, create_token_without_org_role
+from src.backend.lib.logging_config import log_format
+
+# Set up logging configuration
+logging.basicConfig(level=logging.INFO, format=log_format)
+logger = logging.getLogger(__name__)
+
+
+class TestFileListAPI(unittest.TestCase):
+    BASE_URL = f"http://{os.getenv('CHAT_SERVICE_HOST')}:{os.getenv('CHAT_SERVICE_PORT')}"
+
+    def setUp(self):
+        """Setup function to initialize customer, token, and upload files."""
+        logger.info("=== Starting setup process ===")
+
+        # Initialize customer and token
+        customer_data = add_customer("test_org")
+        self.org_id = customer_data.get("org_id")
+        self.token = create_test_token(org_id=self.org_id, org_role="org:admin")
+        self.headers = {'Authorization': f'Bearer {self.token}'}
+
+        # Upload multiple files for the customer
+        self.upload_files(5)  # Upload 5 files
+        logger.info("=== Setup process completed ===")
```

Comment:
Add which testcase it is executing in this line in setup function

And add which testcase has finished exection in tearDown function

Check with Bharani - he has written it for his testcases

Write it for both the test files